pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -683,7 +691,8 @@ def new_event_body():
             elif isinstance(part, TextPart):
                 if body.get('content'):
                     body = new_event_body()
-                body['content'] = part.content
+                if settings.include_content:
+                    body['content'] = part.content
```

Comment:
This implies that the number of events produced will depend on whether content is included if multiple tool calls are made in a single message.

Reply:
Yep, I get it. Something like this would cause this issue. 

```
    response = ModelResponse(parts=[
        TextPart('Some text response'),
        ToolCallPart('tool2', {'arg2': 'value2'}, 'call_2'),
        TextPart('Some more text response'),
        ToolCallPart('tool1', {'arg1': 'value1'}, 'call_1'),
        TextPart('Even more text response'),
    ])
```